2nd-gen/packages/swc/components/asset/stories/asset.stories.ts

```diff
@@ -68,42 +62,62 @@ export default meta;
 // ────────────────────
 
 export const Playground: Story = {
-    render: (args) => template(args),
     args: {
         label: 'Background',
         'default-slot': `<img src="https://picsum.photos/id/56/80/80/?blur=2" alt="preview of background" />`,
     },
     tags: ['autodocs', 'dev'],
 };
 
+// ────────────────────
+//    OVERVIEW STORY
+// ────────────────────
+
+export const Overview: Story = {
+    args: {
+        label: 'Background',
+        'default-slot': `<img src="https://picsum.photos/id/56/80/80/?blur=2" alt="preview of background" />`,
+    },
+    tags: ['overview'],
+};
+
 // ──────────────────────────
 //    ANATOMY STORIES
 // ──────────────────────────
 
-const anatomyArgs = [
-    {
-        variant: 'file',
-        label: 'packages/swc/',
-    },
-    {
-        label: 'Avatar',
-        'default-slot': `<img src="https://picsum.photos/id/64/80/80" alt="preview of the avatar" />`,
-    },
-];
 /**
- * An asset is made up of the following parts:
+ * ### Visual structure
+ *
+ * An asset consists of:
+ *
+ * 1. **Icon or image content** - Either a file/folder icon or custom slotted content
+ * 2. **Accessible label** - Provides context for assistive technologies
+ *
+ * The asset automatically centers its content both horizontally and vertically within the available space.
+ *
+ * ### Technical structure
+ *
+ * #### Slots
+ *
+ * - **Default slot**: Custom content to display (typically an image) when variant is not set
+ *
+ * #### Properties
  *
- * - A large file or folder icon based on the asset `variant`
- * - An accessible label for the asset
- * - Optional content to be displayed in the asset when an acceptable value for `variant` is not present
+ * Properties that control the asset's appearance:
+ *
+ * - [**Variant**](#variants): Controls which built-in icon to display (`file`, `folder`, or unset for custom content)
+ * - **Label**: Accessible label for screen readers (used as `aria-label` on the icon SVGs)
  */
 export const Anatomy: Story = {
     render: (args) => html`
-        ${anatomyArgs.map((arg) => template({ ...args, ...arg }))}
+        ${template({ ...args, variant: 'file', label: 'README.md' })}
+        ${template({ ...args, variant: 'folder', label: 'packages/swc/' })}
+        ${template({
+            ...args,
+            label: 'User avatar',
+            'default-slot': `<img src="https://picsum.photos/id/64/80/80" alt="User avatar preview" />`,
```

Comment:
Label and image`alt` should not be redundant:

```suggestion
            label: 'images/profile_sm.png',
            'default-slot': `<img src="https://picsum.photos/id/64/80/80" alt="Headshot of Jenn" />`,
```